alternator/streams.cc

```diff
@@ -702,21 +791,19 @@ future<executor::request_return_type> executor::get_shard_iterator(client_state&
 }
 
 struct event_id {
-    cdc::stream_id stream;
+    utils::UUID stream;
     utils::UUID timestamp;
 
     static const auto marker = 'E';
 
-    event_id(cdc::stream_id s, utils::UUID ts)
+    event_id(utils::UUID s, utils::UUID ts)
         : stream(s)
         , timestamp(ts)
     {}
     
     friend std::ostream& operator<<(std::ostream& os, const event_id& id) {
         boost::io::ios_flags_saver fs(os);
```

Comment:
You don't need flags_saver any more because you removed the std::hex usage.

Reply:
true